source/includes/api/recognition/_nominations.md

```diff
@@ -0,0 +1,58 @@
+## Nominations
+
+### Update a Nomination's status
+
+An endpoint to update a status of a nomination for a given campaign. By passing
+a status parameter. An approver for the nomination must also be provided.
+You will be returned the updated JSON for the given nomination.
+
+``` http
+PATCH /api/v2/campaigns/:campaign_id/nominations/:id HTTP/1.1
+Authorization: Token token=xxx
+Content-Type: application/json
+
+{
+  "transition" : "approve",
+  "approver" : 123
+}
+```
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id" : "1",
+  "state" : "approved_l1",
+}
+```
+
+#### HTTP Request
+
+`PATCH /api/v2/campaigns/:campaign_id/nominations/:id`
+
+#### Attributes
+
+Attribute | Type | Info
+--------- | ---- | ----
+id | integer | (required) nomination id
```

Comment:
Split out URL attributes from body attributes? I believe there's an example of this elsewhere in the docs